### DISCUSSION TITLE
moving I2C pins ?

### DISCUSSION THREAD

**Discussion by herbkonik** [dbuezas/lgt8fx#220] [General]
Hi,
is it posible to move the I2C pins to some other pins (E.g. pin 11 and 12) ?
Reason is (what a surprise... ;-) ) i need a few analog pins more, - if possible at the Nano Style bord.
I know there is a  LQFP48 Board (i allready ordered a few) but of corse the size and the pinout is different and i have a few circuits, made for the Nano, where it also would not be bad to have 2 analog inputs more.

**Comment by trimarco232** [comment]
Hi,
looking at the PMX0, PMX1 and PMX3 Registers, you have no luck
time to try a software i2c , like SoftI2CMaster

**Comment by herbkonik** [comment]
SoftI2CMaster i've  tried already but don't get a solution.
Today i changed to U8g2 Lib for the display, which uses software i2c by default, - works great and it needs less menory (depends of the Font).

Dosn't LGT8Fx not use sotware I2C by default ? Thought to read it somewhere...
